grpc/server/src/backend.h

```diff
@@ -1,28 +1,22 @@
-#include <grpc++/server.h>
 #include <memory>
-#include <mutex>
-
-#include "connection_initiator.h"
-#include "dronecore.h"
 
 namespace dronecore {
 namespace backend {
 
 class DroneCoreBackend
 {
 public:
-    DroneCoreBackend() {}
-    ~DroneCoreBackend() {}
+    DroneCoreBackend();
+    ~DroneCoreBackend();
+
+    DroneCoreBackend(DroneCoreBackend &&op) noexcept;
```

Comment:
Why is the `noexcept` needed? Shouldn't everything be compiled without exception anyway?

Reply:
Not, sure, I copied a pImpl example to not mess it up. Should I remove it?

Reply:
I guess so, not sure.